### DISCUSSION TITLE
Help with exports

### DISCUSSION THREAD

**Discussion by potato25** [nesbox/TIC-80#1560] [Q&A]
How do i export html game ? 
after i get zip file , 
I Extract it and i get : index.html , tic80.js , tic80.wasm and cart.tic

index.html doesn't run in Google Chrome or Microsoft Explorer

I just want game html file

**Comment by MFauzan26** [comment]
you can't run the html file directly, you must to use a local server to run it, or you can upload it on itch io.

**Reply by potato25** [reply]
> you can't run the html file directly, you must to use a local server to run it, or you can upload it on itch io.

Oh , i see 👍
